io-engine/src/core/snapshot.rs

```diff
@@ -62,6 +95,27 @@ pub trait SnapshotDescriptor {
     fn name(&self) -> Option<String>;
 }
 
+/// Traits gives ListSnapshot Parameters.
+pub trait ListSnapshotParameters {
+    /// Get SnapshotParameters.
+    fn snapshot_params(&self) -> SnapshotParams;
+
+    /// Get snapshot_uuid.
+    fn snapshot_uuid(&self) -> String;
```

Comment:
You can avoid creating a copy of String by using '&str'.

Reply:
> You can avoid creating a copy of String by using '&str'.

Accepted, Code changed.